src/main/java/com/couchbase/lite/Database.java

```diff
@@ -1347,6 +1347,9 @@ public RevisionList unpushedRevisionsSince(String sequence,
 
     @InterfaceAudience.Private
     public Map<String, Object> getAllDocs(QueryOptions options) throws CouchbaseLiteException {
+
+        // TODO - This method is not fully implemented. Further work will be required.
```

Comment:
Do we have an issue # for this? Could you please reference it here if we have one?